src/main/scala/org/broadinstitute/dsde/firecloud/service/OAuthService.scala

```diff
@@ -122,4 +150,12 @@ trait OAuthService extends HttpService with PerRequestCreator with FireCloudDire
       }
     }
 
+
+  def initiateAuth(state: String, approvalPrompt: String) =  {
+    val gcsAuthUrl = HttpGoogleServicesDAO.getGoogleRedirectURI(state, approvalPrompt=approvalPrompt)
+    redirect(gcsAuthUrl, StatusCodes.TemporaryRedirect)
+  }
```

Comment:
this was factored out into its own method to support future functionality; it's overkill for this PR, but I'll be using this in other places soon.